pkg/harvester/l10n/en-us.yaml

```diff
@@ -495,6 +500,10 @@ harvester:
         ShutdownAndRestartAfterEnable: Shutdown and Restart After Enable
         ShutdownAndRestartAfterDisable: Shutdown and Restart After Disable
         Shutdown: Shutdown
+    cpuPinning:
+      label: Enable CPU Pinning
+      tooltip: Enable CPU Pinning brings better performance and reduce latency for the virtual machine 
```

Comment:
@brandboat , see if this tooltip is okay for you or have any suggestion ?

![Uploading image.png…]()